soc/arm/nordic_nrf/nrf53/soc.c

```diff
@@ -169,7 +163,7 @@ static int nordicsemi_nrf53_init(const struct device *arg)
 	nrf_regulators_dcdcen_vddh_set(NRF_REGULATORS, true);
 #endif
 
-#if defined(NRF_GPIO_FORWARDER_FOR_NRF5340_CPUAPP_ENABLED)
+#if defined(CONFIG_BOARD_NRF_GPIO_FORWARDER_FOR_NRF5340)
```

Comment:
The `BOARD_NRF_GPIO_FORWARDER_FOR_NRF5340` option is defined as available only for `nrf5340dk_nrf5340_cpuapp*` targets, but also others, like `thingy53_nrf5340_cpuapp*` and `bl5340_dvk_cpuapp*`, need to use the code wrapped in this `#if`.

Reply:
Thanks. I moved the Kconfig to SOC common, since it affects nrf53/soc.c, and then all boards need to select it on their BOARD_ENABLE_CPUNET.